starters/solidjs-tanstack-tailwind/.prettierrc

```diff
@@ -0,0 +1,4 @@
+{
+  "trailingComma": "es5",
```

Comment:
This is the default value and can be omitted.

Reply:
Thanks!!